### PR TITLE
docs: fix useAnimatedKeyboard page crash

### PR DESCRIPTION
**Pull request by patrycjakalinska**
Due to change in InteractiveExample, `useAnimatedVideo` page on production crashed. To fix that we transformed `useAnimatedVideo` InteraciveExample component to a functional component.

Before:
image

After:
image

### CODE REVIEW

File: docs/docs/device/useAnimatedKeyboard.mdx [hunk 1]
```diff
@@ -79,20 +79,9 @@ Hook `useAnimatedKeyboard` returns an object containing these fields:
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 import AnimatedKeyboardSrc from '!!raw-loader!@site/src/examples/AnimatedKeyboard';
+import AnimatedKeyboard from '@site/src/examples/AnimatedKeyboardVideoComponent';
```
Comment (piaskowyk) [inline docs/docs/device/useAnimatedKeyboard.mdx:82]:
```suggestion
import AnimatedKeyboardVideo from '@site/src/examples/AnimatedKeyboardVideoComponent';
```

File: docs/src/examples/AnimatedKeyboardVideoComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import ThemedVideo from '../components/ThemedVideo';
+
+const App: React.FC<{ width?: number }> = () => (
```
Comment (piaskowyk) [inline docs/src/examples/AnimatedKeyboardVideoComponent.tsx:4]:
```suggestion
const AnimatedKeyboardVideo: React.FC<{ width?: number }> = () => (
```

File: docs/src/examples/AnimatedKeyboardVideoComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+    center
+    width={300}
+    sources={{
+      light: '/recordings/useAnimatedKeyboard_light.mov',
+      dark: '/recordings/useAnimatedKeyboard_dark.mov',
+    }}
+  />
+);
+
+export default App;
```
Comment (piaskowyk) [inline docs/src/examples/AnimatedKeyboardVideoComponent.tsx:15]:
```suggestion
export default AnimatedKeyboardVideo;
```

**Review by piaskowyk — COMMENTED** [review]
Just some renames

**Review by m-bert — COMMENTED** [review]
Looks good 😄 Please address the comments by @piaskowyk, and once that's done, we are good to go :shipit:

**Review by kacperkapusciak — COMMENTED** [review]
we can fix it way simpler

```jsx
<InteractiveExample
  src={AnimatedKeyboardSrc}
  component={() => (
    <ThemedVideo
      center
      width={300}
      sources={{
        light: '/recordings/useAnimatedKeyboard_light.mov',
        dark: '/recordings/useAnimatedKeyboard_dark.mov',
      }}
    />
  )}
/>
```

There_Is_Another

**Review by kacperkapusciak — APPROVED** [review]
🚀

**Pull request merged**